testcases/multizone_large_office_eplus/doc/index.html

```diff
@@ -0,0 +1,317 @@
+</li>
+<li>
+<code>EPlus96_reaPeoCouTopEas_y</code> [1] [min=None, max=None]: Number of people in east zone on top floor
+</li>
+<li>
+<code>EPlus96_reaPeoCouTopNor_y</code> [1] [min=None, max=None]: Number of people in north zone on top floor
+</li>
+<li>
+<code>EPlus96_reaPeoCouTopSou_y</code> [1] [min=None, max=None]: Number of people in south zone on top floor
+</li>
+<li>
+<code>EPlus96_reaPeoCouTopWes_y</code> [1] [min=None, max=None]: Number of people in west zone on top floor
+</li>
+<li>
+<code>EPlus96_reaRelHum_y</code> [1] [min=None, max=None]: Relative humidity
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaBotCor_y</code> [W] [min=None, max=None]: Sensible cooling load in core zone on bot floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaBotEas_y</code> [W] [min=None, max=None]: Sensible cooling load in east zone on bot floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaBotNor_y</code> [W] [min=None, max=None]: Sensible cooling load in north zone on bot floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaBotSou_y</code> [W] [min=None, max=None]: Sensible cooling load in south zone on bot floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaBotWes_y</code> [W] [min=None, max=None]: Sensible cooling load in west zone on bot floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaMidCor_y</code> [W] [min=None, max=None]: Sensible cooling load in core zone on mid floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaMidEas_y</code> [W] [min=None, max=None]: Sensible cooling load in east zone on mid floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaMidNor_y</code> [W] [min=None, max=None]: Sensible cooling load in north zone on mid floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaMidSou_y</code> [W] [min=None, max=None]: Sensible cooling load in south zone on mid floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaMidWes_y</code> [W] [min=None, max=None]: Sensible cooling load in west zone on mid floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaTopCor_y</code> [W] [min=None, max=None]: Sensible cooling load in core zone on top floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaTopEas_y</code> [W] [min=None, max=None]: Sensible cooling load in east zone on top floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaTopNor_y</code> [W] [min=None, max=None]: Sensible cooling load in north zone on top floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaTopSou_y</code> [W] [min=None, max=None]: Sensible cooling load in south zone on top floor
+</li>
+<li>
+<code>EPlus96_reaSenCooLoaTopWes_y</code> [W] [min=None, max=None]: Sensible cooling load in west zone on top floor
+</li>
+<li>
+<code>EPlus96_reaTDryBul_y</code> [K] [min=None, max=None]: Drybulb temperature
+</li>
+<li>
+<code>EPlus96_reaTWetBul_y</code> [K] [min=None, max=None]: Wetbulb temperature
+</li>
+</ul>
+<h3>Additional System Design</h3>
+<h4>Lighting</h4>
+<p>
+Lighting heat gain is included in the internal heat gains and is not controllable.
+</p>
+<h4>Shading</h4>
```

Comment:
Are "Shading"  and ">Onsite Generation and Storage" required. If not, consider to remove them